metaflow/plugins/argo/argo_workflows.py

```diff
@@ -187,13 +187,19 @@ def _sanitize(name):
     def list_templates(flow_name, all=False):
```

Comment:
rather than use `flow_name`, we can rely on `self.flow.name` - tad bit more safe